### ISSUE TITLE
Clicking User details shows  Not Found (probably missing DB update?)

### ISSUE THREAD

**Issue by benapetr** [ansible/awx#9238]
##### ISSUE TYPE
 - Bug Report

##### SUMMARY
After upgrading from AWX 11 to 17.0.1 it seems that there are some updates to DB schemas missing, when I click user details in top menu, it throws me to "old skin" AWX error, see screenshot

##### ENVIRONMENT
* AWX version: 17.0.1
* AWX install method: docker on linux
* Ansible version:  X.Y.Z
* Operating System: CentOS 7
* Web Browser: Firefox

##### STEPS TO REPRODUCE

Upgrade from AWX 11 to 17.0.1

##### EXPECTED RESULTS

I would expect link to work and show user details?

##### ACTUAL RESULTS

<img width="622" alt="image" src="https://user-images.githubusercontent.com/1560121/106906401-6c9e3a00-66fd-11eb-973f-eb789736b955.png">

**Comment by jakemcdermott** [comment]
> I would expect link to work and show user details?

I agree. That said, I tried to reproduce this but was unsuccessful:

![user-link](https://user-images.githubusercontent.com/9753817/106919037-c12ebe80-66d7-11eb-9d86-4b13499f4d22.gif)

It could be a sign that the 11->17 upgrade, data migration, etc. didn't finish successfully like you say but without more information it isn't really something we can troubleshoot over github.

Have you tried the link using a completely fresh install of v17? It might help rule out an upgrade issue if you still encounter the bug.

**Comment by MrBones757** [comment]
I saw this issue with my user account sourced from LDAP/AD in the instance i have access to after the 17.0.1 upgrade when navigating from a job history details page and clicking on the launch user (We upgraded 14.1.0 -> 15.0.1 -> 16.0.0 -> 17.0.1 on kubernetes) however i saw this issue only once and it seems to have resolve its self. I have not been able to replicate this issue again (perhaps it fixes itself with re-authentication in this case or something)

**Comment by uejo** [comment]
same issue here, 
migrated from 14 -> 17.0.1
when i click on user details, i get directed to localhost/users/5/details
to correctly display the user details i can manually enter localhost/#/users/5/details.

**Comment by unlikelyzero** [comment]
Hey all, this is a duplicate of https://github.com/ansible/awx/issues/8998 which was not included in the 17.0.1 release. Please stay on the lookout for the next release of awx!
